packages/manager/apps/telecom/src/app/telecom/telephony/service/contact/contact.service.js

```diff
@@ -0,0 +1,99 @@
+import map from 'lodash/map';
+import set from 'lodash/set';
+
+export default class {
+  /* @ngInject */
+  constructor($http, $q, $stateParams) {
+    this.$http = $http;
+    this.$q = $q;
+    this.$stateParams = $stateParams;
```

Comment:
**suggestion**: Instead of using `$stateParams` directly. The required values can be passed from the caller and make the service reusable.

Reply:
it's apply